docs/application/vscode-ext/Tizen/hybrid.md

```diff
@@ -0,0 +1,223 @@
+# Hybrid application development
+
+The VS code extension installation can be done in two different ways. Refer to [VS code extension installation](dotnet.md) section for details. 
+
+## Develop Hybrid application
+
+The following sections explain how to use Visual Studio Code Extension for Tizen to develop your hybrid applications.
+
+### Create Tizen Native project
+
+To create a Tizen Native project: Refer to [Create Tizen Native project](native.md)
+
+### Create Tizen Dotnet project
+
+1. Open the **Command Palette** and select **Tizen: Create Tizen project**.
+
+   ![Create project](media/dotnet_project_create.png)
+
+2. Select the project type as **Dotnet**.
+
+   ![Select project](media/dotnet_project_select.png)
+
+3. Select the required profile for your application development.
+
+   ![Select profile](media/dotnet_profile.png)
+
+4. Select the profile version for your application.
+
+   ![Select version](media/dotnet_profile_version.png)
+
+5. Select the required project template(eg:TizenNUIapp) for your application from template list.
+
+   ![Select template](media/dotnet_project_template.png)
+
+6. Enter the project name.
+
+   ![project name](media/dotnet_project_name.png)
+
+7. If prompts for setting current project as working folder, select Yes or No
+
+   ![project name](media/dotnet_project_working_folder.png)
+
+### Select Working Folder 
+
+1. Open the **Command Palette** and select **Tizen: Set Tizen Working Folder**.
+
+   ![Set working folder](media/set_working_folder.png)
+
+2. Select the working folder of tizen.
+
+   ![Select working folder](media/select_working_folder_project.png)
+
+3. Selected working folder will be updated in workspace.yaml file
+
+   ![Selected working folder](media/selected_working_folder_workspace_yaml.png)
+
+Selected working folder will be considered as **main app**.
+
+### Update Dependency projects
+
+1. From vscode File explorer Right click on **main app** and select **Tizen:Update Dependency Projects**.
+
+   ![Set dependency projects](media/select_update_dependency_project.png)
+
+2. Quick pick list with project list for selection will be displayed.
+
+   ![Display depencency project list](media/select_dependency_project.png)
+
+3. Select the dependency projects and press OK button.
+
+   ![Select dependency project](media/selected_dependency_project.png)
+
+4. Selected dependencies will be updated in workspace.yaml file
+
+   ![Selected dependencies projects](media/selected_dependency_workspace_yaml.png)
+
+Hybrid app successfully created with selected working folder as main app and selected  dependency projects to main app as sub apps.

```

Comment:
Fixed grammatical errors, added an article before 'Hybrid App' and added a comma to improve readability.
```suggestion
The Hybrid app is successfully created with selected working folder as main app, and selected dependency projects to main app as sub apps.

```